lib/theme_store/git_importer.rb

```diff
@@ -109,49 +108,53 @@ def clone_args(config = {})
     args << "clone"
 
     if @branch.present?
-      args.concat(["--single-branch", "-b", @branch])
```

Comment:
Why did you remove `--single-branch`? It seems like a nice optimization. Would it make sense to have something like `--depth 1` to optimize cloning?

At first thought, I think it is not very important because discourse themes probably are quite small, except for those which have many assets, in which case they are probably quite big and not a very good idea to fetch whole history.